config/database.rb

```diff
@@ -1,13 +1,35 @@
 configure do
-  # Log queries to STDOUT in development
-  if Sinatra::Application.development?
+
+  case Sinatra::Application.environment
```

Comment:
Rather than a `case` here why not use `configure :development do ... end`, etc?

The `sinatra/activerecord` support also recognizes `config/database.yml` that works just like Rails... so that could be cleaner, but maybe showing the programmatic way is useful for students, not sure...?